Xamarin.Forms.Platform.Android/FastRenderers/FrameRenderer.cs

```diff
@@ -219,6 +220,13 @@ protected override void OnLayout(bool changed, int left, int top, int right, int
 			_hasLayoutOccurred = true;
 		}
 
+		public override void Draw(Canvas canvas)
```

Comment:
Do we have a measure for how this is going to affect performance? (same question for all new `override Draw` instances)

Reply:
I have added a new sample in Core Gallery.
![clip-performance](https://user-images.githubusercontent.com/6755973/84058510-a295ed80-a9b9-11ea-8b0b-7c3e6d2daa7c.gif)
The sample load hundreds of images first and then the same images using `EllipseGeometry` to Clip it.
The results from iOS are the following (Debug Mode):
- 100 images: 18.82ms slower using Clip.
- 1000 images: 212.87ms slower using Clip.

On Android, the differences are lower.

Reply:
I need to think about Custom Effects too, it can have an impact.